kvui.py

```diff
@@ -386,6 +386,60 @@ def insert_text(self, substring, from_undo=False):
         return super(ConnectBarTextInput, self).insert_text(s, from_undo=from_undo)
 
 
+def is_command_input(string: str) -> bool:
+    return len(string) > 0 and string[0] in "/!"
```

Comment:
Could this be `return string and string[0] in "/!"` or is that too dank lol

Reply:
...yeah nvm that is probably a level of Python abuse that is too dank for mortals to comprehend

~~It would catch "None" as well though, kind of~~

Reply:
The typing would be wrong, because it wouldn't return `bool` for an empty string.

Reply:
Yeah the line I'm drawing is whether mypy highlights it as an error. Earlier I had Vi's exact proposal and beauxq (and mypy) rightly pointed out the return type on that is `str | bool` rather than `bool`